src/ui/Views/Organisation/About.cshtml

```diff
@@ -1,7 +1,7 @@
-@model OrganisationViewModel
+@model OrganisationViewModel;
 
 @{
-  ViewBag.Title = Model.InstitutionName;
+  ViewBag.Title = "About " + Model.InstitutionName;
```

Comment:
Can the two pages be:
About your organisation

And:
Edit ‘About your organisation’